bots/pr/src/main/java/org/openjdk/skara/bots/pr/IntegrateCommand.java

```diff
@@ -61,6 +61,9 @@ private Optional<String> checkProblem(Map<String, Check> performedChecks, String
         final var inProgress = "the status check `" + checkName + "` is still in progress";
         final var outdated = "the status check `" + checkName + "` has not been performed on commit %s yet";
 
+        checkName = checkName.equals("jcheck") && pr.repository().forge().name().equals("GitHub") ?
```

Comment:
There is just one caller of this method, which explicitly calls with `"jcheck"`. The caller should generate the new check name instead of rewriting it here.

Reply:
But in that way, the checkName will be "jcheck-repo#pull" and the error message showed to user would be "the status check jcheck-repo#pull is still in progress".

Reply:
I think that's what we want. That's the name the check will have in the GitHub UI as well.